ibis/pandas/aggcontext.py

```diff
@@ -326,6 +327,83 @@ def __init__(self, kind, *args, **kwargs):
         )
         self.construct_window = operator.methodcaller(kind, *args, **kwargs)
 
+    def _agg_built_in(self, frame, windowed, function, *args, **kwargs):
```

Comment:
I would make both of these module level functions (don't pass self, I think you can just pass max_lookback as as kord arg).

Reply:
I feel that these methods are very particular to Window aggregation context and therefore probably belongs in the class. I curious why module level functions are better here?

Reply:
Discussed offline. Moved to module level.